### PR TITLE
Translator for theses.fr

### PR DESCRIPTION
**Pull request by TesteFixe**
A new translator for http://theses.fr, institutional repository of french doctoral thesis

### CODE REVIEW

File: ThesesFR.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+				"university": "Lyon 1",
+				"url": "http://www.theses.fr/2012LYO10182",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					{
+						"tag": "RGD Chitosan Nanoparticles Cell adhesion Wound healing"
```
Comment (adam3smith) [inline ThesesFR.js]: These don't look right. These are multiple tags. Maybe split on capital letter?

File: ThesesFR.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+function scrape(doc, url) {
+
+	var translator = Zotero.loadTranslator('web');
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48'); //https://github.com/zotero/translators/blob/master/Embedded%20Metadata.js
+	translator.setDocument(doc);
+	translator.setHandler('itemDone', function(obj, item) {
+		//add type
+		//converting internal info:eu-repo/semantics/doctoralThesis notation to "Ph.D. Thesis" only for collections and items
+		if (item.url.indexOf("/handle/") > -1) {
```
Comment (adam3smith) [inline ThesesFR.js]: use `includes("string")`

File: ThesesFR.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+ 
+	if (url.indexOf("?q=") != -1) {
```
Comment (adam3smith) [inline ThesesFR.js]: use `includes()` for testing strings

File: ThesesFR.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+		return "multiple";
+	} else if (url.indexOf("?q=") == -1) {
```
Comment (adam3smith) [inline ThesesFR.js]: ditto

Comment (adam3smith) [reply ThesesFR.js]: also, take advantage of `checkSearchResults` below and use that here to avoid false positives.

Comment (TesteFixe) [reply ThesesFR.js]: Hi,
Theses.fr is Ok : link works : http://www.theses.fr/s197772
Thanks a lot for your reviewing : i'll make the changes asap!

**Comment by symac** [comment]
Hi @adam3smith, do you still have this issue ? Right now if I follow the link you gave, I can see a detailed record : 
capture du 2018-08-23 17-11-32

**Comment by adam3smith** [comment]
@symac working now -- I believe the issue wasn't just for me

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**